src/index.ts

```diff
@@ -401,13 +407,16 @@ export class MetadataComponent extends BaseComponent {
         const $metadataGroup: JQuery = this._buildMetadataGroup(metadataGroup);
 				this._$metadataGroups.append($metadataGroup);
 				const $value: any = $metadataGroup.find(".value");
+				const $items: any = $metadataGroup.find(".item");
 
         if (this._data.limit && this._data.content) {
           if (this._data.limitType === LimitType.LINES) {
-            $value.toggleExpandTextByLines(
+            $items.toggleExpandTextByLines(
```

Comment:
Yeah it's pretty convoluted! That function is defined in the UV repo JQueryPlugins, I've just made a UV PR which is in tandem with this one and might make things clearer: https://github.com/UniversalViewer/universalviewer/pull/1148

Reply:
Interesting... that sounds to me like a pre-existing bug in this component, since if this code depends on a custom plugin defined by UV, then it's not as much of a stand-alone component as I think was intended. I wonder if we need to do some refactoring so that the UV can inject that logic instead of having it hard-coded here, to make this more general-purpose/reusable. Obviously that's a task for a separate PR, but something to consider for the future!

Reply:
I agree, that would make things much tidier. Because they're separated I wasn't getting typescript hints either.

Maybe of interest: the iiif-metadata-component has an examples/index.html, useful for testing. That imports the relevant JQueryPlugins in the header, so it's also available from here:
`script src="https://unpkg.com/@edsilv/jquery-plugins@1.0.7/dist/jquery-plugins.js"></script>`

Reply:
Ahh, I see, and that's coming from this GitHub repo: https://github.com/edsilv/jquery-plugins -- so that raises the question of whether further maintenance work is needed there! Aargh.

Reply:
Obviously, the long-term plan is to get rid of jQuery entirely. I wonder if revising code to avoid the need for this plugin at all would be a helpful next step... but then we have to figure out what to replace it with. Never simple!